bindata/network/multus/multus.yaml

```diff
@@ -31,13 +31,22 @@ spec:
       initContainers:
       - name: cni-plugins
         image: {{.CNIPluginsImage}}
-        command: ["/bin/sh"]
-        args: ["-c", "cp -rf /usr/src/plugins/bin/* /host/opt/cni/bin"]
+        command: ["/bin/copycni.sh"]
         securityContext:
           privileged: true
         volumeMounts:
         - mountPath: /host/opt/cni/bin
           name: cnibin
+        - mountPath: /host/etc-os-release
```

Comment:
I wouldn't do it this way. If you mount `/etc/os-release` in to the container, then crio will follow the symlinks and it will Just Work.

Just make sure the hostPath volume is `type: File`

Reply:
Thanks for the tip on the `type: File` -- updated (and removed the alternate method to look at both locations)